lib/timescaledb/cleaner.ex

```diff
@@ -32,26 +33,34 @@ defmodule Membrane.RTC.Engine.TimescaleDB.Cleaner do
     apply(GenServer, function, [__MODULE__, reporter_options, gen_server_options])
   end
 
-  @impl true
-  def init(args) do
-    state = Map.new(args)
+  @spec stop(cleaner()) :: :ok
+  def stop(cleaner) do
+    GenServer.cast(cleaner, :stop)
```

Comment:
Do you really need a async version of stop? Otherwise you could simply use [`GenServer.stop`](https://hexdocs.pm/elixir/1.13.4/GenServer.html#stop/3)